radixdlt-java/src/test/java/com/radixdlt/client/core/address/RadixAddressTest.java

```diff
@@ -22,7 +23,7 @@ public void createAddressFromPublicKey() {
 		assertEquals(address, RadixAddress.from("JHB89drvftPj6zVCNjnaijURk8D8AMFw4mVja19aoBGmRXWchnJ"));
 	}
 
-	@Test(expected = IllegalArgumentException.class)
```

Comment:
new version of library throwing `DecoderException` instead of `IllegalArgumentException` in case of invalid key